pkg/resources/grant_ownership_acceptance_test.go

```diff
@@ -0,0 +1,491 @@
+package resources_test
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
```

Comment:
please add a few negative tests with bad configs.

Also, the test if something happens outside of terraform (but I think such a test will be added in the followup PR with more sophisticated edge cases).

Reply:
Yeah, added to the list in the pr description to overall test outside of terraform interactions.